applications/FluidDynamicsApplication/python_scripts/navier_stokes_embedded_solver.py

```diff
@@ -247,6 +247,24 @@ def _SetEmbeddedElementReplaceSettings(self):
             else:
                 raise Exception("Domain size is not 2 or 3!!")
 
+        elif (solver_type == "EmbeddedDevelopment"):
+            if(domain_size == 3):
+                self.settings["element_replace_settings"] = KratosMultiphysics.Parameters("""
+                {
+                    "element_name":"EmbeddedSymbolicNavierStokes3D4N",
+                    "condition_name": "Condition3D3N"
```

Comment:
This Should be SurfaceCondition3D3N, also i think there is a special condition for this.

@rubenzorrilla comments?

Reply:
There is a condition for this formulation (NavierStokesWallCondition3D3N / NavierStokesWallCondition2D2N) which adds the traction boundary terms in case there is a Neumann BC.